src/job/jobHandler.ts

```diff
@@ -342,6 +342,7 @@ export abstract class JobHandler {
     const snootyFrontEndVars = {
       GATSBY_BASE_URL: this._config.get<string>('gatsbyBaseUrl'),
       PREVIEW_BUILD_ENABLED: this._config.get<string>('previewBuildEnabled'),
+      FEATURE_FLAG_SEARCH_UI: this._config.get<string>('featureFlagSearchUI'),
```

Comment:
minor nitpick. would prefix with `GATSBY_` but not a deal breaker

Reply:
ahhh, I see this in the documentation now. I think that's reasonable. How do we feel about this (FE) being `GATSBY_TEST_SEARCH_UI` while the rest of the pipeline of the variable (BE) stays `FEATURE_FLAG_SEARCH_UI`? Should I change all of the naming to match?

Reply:
Oh, sure! https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/